From b3fc2ac3b2ae18a1aa962fa1091cb5257323e629 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Fri, 25 Sep 2020 11:28:01 +0800
Subject: [PATCH] ARM: configs: rv1126-battery.config: rework by standard
 process

Update by:
make ARCH=arm rv1126_defconfig rv1126-tb.config
cp .config rv1126.config
make ARCH=arm rv1126_defconfig rv1126-tb.config rv1126-battery.config
scripts/diffconfig -m rv1126.config .config > arch/arm/configs/rv1126-battery.config

Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
Change-Id: I9f24066c7bb3679898b81e3229715c964a000856
---
 arch/arm/configs/rv1126-battery.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/rv1126-battery.config b/arch/arm/configs/rv1126-battery.config
index 1956878f2dca..4fc20c3bee9c 100644
--- a/arch/arm/configs/rv1126-battery.config
+++ b/arch/arm/configs/rv1126-battery.config
@@ -1,4 +1,4 @@
 CONFIG_BATTERY_CW2015=y
 CONFIG_POWER_RESET_GPIO=y
 CONFIG_RTC_DRV_PCF8563=y
-CONFIG_SND_SOC_ES8311=y
+CONFIG_SND_SOC_ES8311=m
-- 
2.35.3

